### DISCUSSION TITLE
Student Developer Pack Approval - Location issue

### DISCUSSION THREAD

**Discussion by jairamr26** [community/community#56112] [GitHub Education]
### Hi everyone,

I have been trying to get my student developer pack activated a few times. I could not get it as I keep getting the message that I'm not near any campus location for the school I have selected.
I am in a circumstance where I'm **doing my internship in a different location from my school**, and I **urgently need to activate** this. Is there **any other way to consider this case** and **activate my pack**? I can provide **other proofs at hand** if required.

**Comment by samus-aran** [comment]
Hi @jairamr26 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
